### PR TITLE
Fix Bug 413 'Event URI too long'

### PR DESCRIPTION
**Pull request by yicyao**
Fix Bug 413 'Event URI too long' by passing in eventId instead of full event (when passing in full event, it concats everything about event (description, name, image url which sometimes leads to 8000+ char request params)

### CODE REVIEW

File: project/src/main/java/com/google/sps/servlets/EventRankerServlet.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import com.google.sps.utilities.SpannerTasks;
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.ArrayList;
```
Comment (dash102) [inline project/src/main/java/com/google/sps/servlets/EventRankerServlet.java:12]: nit: alphabetize

File: project/src/main/webapp/shared.js [hunk 2]
```diff
@@ -181,10 +181,15 @@ function togglePrefilledInterests() {
 }
 
 async function getRankedEvents(events) {
-  const response = await fetch('/event-ranker?' + new URLSearchParams({'events': JSON.stringify(events)}));
+  const eventIds = new Array(); 
+  for (index in events) {
```
Comment (dash102) [inline project/src/main/webapp/shared.js:185]: Use the for/of loop instead, for things that are arrays 
https://google.github.io/styleguide/jsguide.html#features-for-loops

File: project/src/main/java/com/google/sps/servlets/EventRankerServlet.java (hunk 1 above)
Comment (yicyao) [reply project/src/main/java/com/google/sps/servlets/EventRankerServlet.java:12]: done

File: project/src/main/webapp/shared.js (hunk 2 above)
Comment (yicyao) [reply project/src/main/webapp/shared.js:185]: yes ty i was looking for something like that!

**Pull request merged**